hnn_core/network.py

```diff
@@ -399,6 +417,51 @@ def __repr__(self):
                  len(self.pos_dict['L5_basket'])))
         return '<%s | %s>' % (class_name, s)
 
+    def _reset_cell_positions(self):
```

Comment:
but wouldn't you need to adjust the `lamtha` too ?

Reply:
indeed, 4517a47 now adjusts `nc_dict['lamtha']` for all connections.